src/views/hostnetworkconfiguration/components/TopologySidebar/InterfaceDrawer/InterfaceDrawer.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC } from 'react';
+import { useLocation } from 'react-router-dom-v5-compat';
+
+import {
+  PageSection,
+  Tab,
+  Tabs as TabsComponent,
+  TabTitleText,
+  Title,
+} from '@patternfly/react-core';
+import { NodeNetworkConfigurationInterface } from '@types';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
+
+import { InterfaceDrawerTabId, InterfaceDrawerTabProps } from './constants';
+import InterfaceDrawerDetailsTab from './InterfaceDrawerDetailsTab';
+import InterfaceDrawerYAMLFooter from './InterfaceDrawerFooter';
+import InterfaceDrawerYAMLTab from './InterfaceDrawerYAMLTab';
+
+type InterfaceDrawerProps = {
+  selectedInterface: NodeNetworkConfigurationInterface;
+};
+
+const InterfaceDrawer: FC<InterfaceDrawerProps> = ({ selectedInterface }) => {
+  const { t } = useNMStateTranslation();
+
+  const location = useLocation();
+
+  const Tabs: InterfaceDrawerTabProps[] = [
```

Comment:
nitpick: `Tabs` -> 'tabs'

Reply:
I don't understand why you've capitalized the variable name. This isn't a component or a type. It's an array of objects.